prow/cmd/branchprotector/protect.go

```diff
@@ -238,28 +239,24 @@ func (p *Protector) UpdateRepo(orgName string, repo string, repoDefaults config.
 	}
 	for _, branch := range branches {
 		bn := branch.Name
-		if err := p.UpdateBranch(orgName, repo, bn, branch.Protected); err != nil {
```

Comment:
If the `Protected` field of `github.Branch` is no longer valid, please remove it from the type.